website/docs/r/log_analytics_datasource_iis_logs.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Log Analytics"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_log_analytics_datasource_iis_logs"
+description: |-
+  Manages a Log Analytics IIS Logs DataSource.
+---
+
+# azurerm_log_analytics_datasource_iis_logs
+
+Manages a Log Analytics IIS Logs DataSource.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-law"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_log_analytics_datasource_iis_logs" "example" {
+  name                = "example-lad-wpc"
+  resource_group_name = azurerm_resource_group.example.name
+  workspace_name      = azurerm_log_analytics_workspace.example.name
+  state               = "OnPremiseEnabled"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Log Analytics IIS Logs DataSource. Changing this forces a new Log Analytics IIS Logs DataSource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Log Analytics IIS Logs DataSource should exist. Changing this forces a new Log Analytics IIS Logs DataSource to be created.
+
+* `workspace_name` - (Required) The name of the Log Analytics Workspace where the Log Analytics IIS Logs DataSource should exist. Changing this forces a new Log Analytics IIS Logs DataSource to be created.
+
+* `state` - (Required) Specifies the start/stop state for collection of performance data from connected IIS servers. Possible values include `OnPremiseEnabled` and `OnPremiseDisabled`.
```

Comment:
```suggestion
* `on_premise` - (Required) Specifies the start/stop state for collection of performance data from connected IIS servers. Possible values include `true` and `false`. 
```

Reply:
Resolved to reflect new schema properties